lib/file_loader.rb

```diff
@@ -0,0 +1,7 @@
+require 'csv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.